metaflow/plugins/aip/aip.py

```diff
@@ -1098,7 +1099,7 @@ def _set_container_labels(self, container_op: ContainerOp):
                 "tags.ledger.zgtools.net/ai-experiment-name", self.experiment
             )
 
-        if ZILLOW_ZODIAC_SERVICE and ZILLOW_ZODIAC_TEAM:
```

Comment:
This looks fine to me, since ZILLOW_ZODIAC_TEAM is not used within the if statement.

I don't have an understanding of how the code works together. Is there any risk if the `add_pod_annotation()` gets called in the case where there is no ZILLOW_ZODIAC_TEAM?

Reply:
I'm marking this PR as draft since I haven't fully worked through it. The changes here do not provide my desired result yet.